MessengerOnFileForLinux/MessageFramework/src/Message.hpp

```diff
@@ -23,6 +23,10 @@ class Message
     Message(std::string messageFlag, std::string username, std::string content);
     Message(std::string fullMessageInRow);
     virtual ~Message() = default;
```

Comment:
przesunąłbym na koniec

Reply:
utrzymujemy konwencję:
c-tors
d-tors
&& c-tor
&& operator= 
& c-tor
& operator

Jest tak w każdym pliku, który ruszałem i staram się to trzymać.